readme.md

```diff
@@ -277,7 +277,9 @@ Setting this to `false` resolves the promise with the error instead of rejecting
 Type: `boolean`<br>
 Default: `true`
 
-Keep track of the spawned process and `kill` it when the parent process exits.
+Kill the spawned process when the parent process exits unless either:
+  - the spawned process is [`detached`](https://nodejs.org/api/child_process.html#child_process_options_detached)
+  - the parent process is terminated abruptly, e.g. with `SIGKILL` (as opposed to `SIGTERM` or a normal exit)
```

Comment:
Don't forget to update the TypeScript definition docs too when doing readme changes.